// Copyright (c) 2022 NodeOps
//
// Distributed under the GNU GPLv3 software license, see the accompanying
// file LICENSE or visit <https://www.gnu.org/licenses/gpl-3.0.en.html>

#ifndef CGNT_REPO_CONFIG_HPP_
#define CGNT_REPO_CONFIG_HPP_

namespace cognitio {
namespace repo {
namespace config {
class Config {};
}  // namespace config
}  // namespace repo
}  // namespace cognitio

#endif  // CGNT_REPO_CONFIG_HPP_
